dts/arm/unisoc/mem.h

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright (c) 2018, UNISOC Incorporated
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef __DT_UNISOC_MEM_H
+#define __DT_UNISOC_MEM_H
+
+#include <mem.h>
+
+#if defined(CONFIG_SOC_UWP5661)
```

Comment:
We've moved away from having file like mem.h here.  Please just merge this into the DTSI files directly.

Reply:
done.